kubernetes-manifests/emailservice.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     spec:
       terminationGracePeriodSeconds: 5
       containers:
-      - name: server
+      - name: emailservice
```

Comment:
is there a specific reason to do this change? if not, please revert for the sake of consistency with other manifests.

Reply:
To show stats aggregated by container name in stackdriver to repro the memory leak. I don't mind changing these back, but a better solution might be to rename all the other containers unless there's a good reason to call them all "server".

<img width="1295" alt="screen shot 2018-12-10 at 11 17 17 pm" src="https://user-images.githubusercontent.com/43799105/49784397-fabf3700-fcd1-11e8-9b58-0e687976d9da.png">

Reply:
Let me know if a33ff61 works for you, if not I'll change them back.

Reply:
Thanks this LGTM.

Reply:
Sorry I meant to say: can we revert all services back to name "server"? :)

Ideally you _really_ should not be inferring the "app name" from "container name". Container name can be just "frontend" or "web" or "server", but doesn't have to actually tell what the app does.

Container name is useful only within the context of a Pod and not across the fleet.